ITAQA/core/Pollutants.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Definition of pollutants
+"""
+
+from enum import Enum, auto
+
+
+class Pollutant(Enum):
+    """Enum holding pollutant types"""
+    # TODO: How do we want to support different measurement types? (8h/24h ...)
```

Comment:
To be decided later on, depending on the data available in all the regions

Reply:
Got it. I'll leave the TODO just to keep it in mind...